src/doc/unstable-book/src/compiler-flags/profile.md

```diff
@@ -0,0 +1,21 @@
+# `profile`
+
+The tracking issue for this feature is: None
```

Comment:
Ah actually can you fill this in with https://github.com/rust-lang/rust/issues/42524?